storybook/pages/RecipientSelectorPanelPage.qml

```diff
@@ -0,0 +1,148 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import AppLayouts.Wallet.adaptors 1.0
+
+import shared.popups.send.panels 1.0
+import shared.stores.send 1.0
+
+import StatusQ.Core.Utils 0.1
+import StatusQ.Core.Theme 0.1
+
+import Storybook 1.0
+import Models 1.0
+
+SplitView {
+    id: root
+
+    ListModel{
+        id: savedAddressesModel
+
+        Component.onCompleted: {
+            const data = []
+            for (let i = 0; i < 10; i++)
+                data.push({
+                              name: "some saved addr name " + i,
+                              ens: [],
+                              address: "0x2B748A02e06B159C7C3E98F5064577B96E55A7b4",
+                              chainShortNames: "eth:arb1"
+                          })
+            append(data)
+        }
+    }
+
+    ListModel {
+        id: recentsModel
+
+        readonly property var data: [
+            {
+                activityEntry:
+                {
+                    sender: "0x10bbfe4072ebb77e53aa9117c7300531d151feaf",
+                    recipient: "0x10bbfe4072ebb77e53aa9117c7300531d151ffff",
+                    timestamp: "1715274859",
+                    txType: 0,
+                    amountCurrency: {
+                        objectName: "",
+                        amount: 1,
+                        symbol:"",
+                        displayDecimals: 0,
+                        stripTrailingZeroes: true
+                    }
+                }
+            },
+            {
+                activityEntry:
+                {
+                    sender: "0x1bbbfe4072ebb77e53aa9117c7300531d151feaf",
+                    recipient: "0xebfbfe4072ebb77e53aa9117c7300531d1511111",
+                    timestamp: "1709832115",
+                    txType: 1,
+                    amountCurrency: {
+                        objectName: "",
+                        amount: 1,
+                        symbol:"",
+                        displayDecimals: 0,
+                        stripTrailingZeroes: true
+                    }
+                }
+            }
+        ]
+
+        Component.onCompleted: append(data)
+    }
+
+    readonly property WalletAccountsAdaptor accountsAdaptor: WalletAccountsAdaptor {
```

Comment:
```suggestion
    WalletAccountsAdaptor {
      id: accountsAdaptor
```

Reply:
I've just removed the adaptor and added an explicit model as per your explanation!